sound/soc/sof/Kconfig

```diff
@@ -50,6 +50,26 @@ config SND_SOC_SOF_DEBUG_PROBES
 	  Say Y if you want to enable probes.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_CLIENT
+	tristate
+	select VIRTUAL_BUS
+	help
+	  This option is not user-selectable but automagically handled by
+	  'select' statements at a higher level
+
+config SND_SOC_SOF_CLIENT_SUPPORT
+	bool "SOF enable clients"
+	depends on SND_SOC_SOF
+	help
+	  This adds support for client support with Sound Open Firmware.
+	  The SOF driver adds the capability to separate out the debug
+	  functionality for IPC tests, probes etc. into separate client
+	  devices. This option would also allow adding some client devices
+	  based on the capabitlies advertised by the DSP during FW boot and
+	  some others based on the ACPI/OF device information.
```

Comment:
"during FW boot, or based on the ACPI/OF device information." 

I'd drop the "some" and "some others", it makes this sound a bit too handwavy, and I don't think it's not, this is adding a mechanism to register a variable set of IPC clients based on FW capabilities and  platform description.

Reply:
done!